airbyte-integrations/connectors/source-google-analytics-v4/acceptance-test-config.yml

```diff
@@ -20,10 +20,12 @@ tests:
       empty_streams: []
       expect_records:
         path: "integration_tests/expected_records.txt"
-  incremental:
-    - config_path: "secrets/service_config.json"
-      configured_catalog_path: "integration_tests/configured_catalog.json"
-      future_state_path: "integration_tests/abnormal_state.json"
+# Since the connector makes 2 days look back window, it can not pass SAT where all records produce cursor value greater ao equal to a state value
```

Comment:
Do we haven't any options to leave incremental tests?

Reply:
Well, I could not find any. The test checks whether the consequent read produces any records with the cursor value less than the one in the state. After this PR it does produce, we do it intentionally to guarantee that all the data is golden (will not be modified in the future)

Reply:
Btw, we have the same situation in [Google Ads](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-google-ads/acceptance-test-config.yml#L31).
As an option, changes to SAT package may be made so that threshold days value is [parametrized](https://github.com/airbytehq/airbyte/issues/12467)